src/main/java/com/booksaw/betterTeams/score/ScoreManagement.java

```diff
@@ -85,15 +85,13 @@ private void sched() {
 
 	@EventHandler
 	public void onPurge(PostPurgeEvent e) {
-		Bukkit.getScheduler().runTask(Main.plugin, () -> {
-			Main.plugin.getConfig().getStringList("purgeCommands").forEach(cmd -> {
-				if (Main.plugin.getServer().getPluginManager().isPluginEnabled("PlaceholderAPI")) {
-					cmd = PlaceholderAPI.setPlaceholders(null, cmd);
-				}
+		Bukkit.getScheduler().runTask(Main.plugin, () -> Main.plugin.getConfig().getStringList("purgeCommands").forEach(cmd -> {
+            if (Main.plugin.getServer().getPluginManager().isPluginEnabled("PlaceholderAPI")) {
```

Comment:
indentation again.

I'll define some form of formatting guide / intellij formatting file when I have some time so imports don't get moved around etc. when different devs make PRs which makes things harder to follow